modules/nodes-cluster-resource-levels-job.adoc

```diff
@@ -61,6 +84,31 @@ $ oc create sa cluster-capacity-sa
 $ oc adm policy add-cluster-role-to-user cluster-capacity-role \
     system:serviceaccount:default:cluster-capacity-sa
 ----
++
+[TIP]
+====
+You can alternatively apply the following YAML to add the role:
+
+.Sample cluster role binding to add the `cluster-capacity-role` to the `cluster-capacity-sa` service account
+[source,yaml]
+----
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cluster-capacity-role
+  uid: 307ffe33-6bfe-4dd8-a236-c4b0e21d3b56
+  resourceVersion: '55140'
+  creationTimestamp: '2021-05-13T20:40:41Z'
```

Comment:
@mikemckiernan Why do we leave the `creationTimestamp`, `resourceVersion` and `uid` in this example, but you explicitly removed them in https://github.com/openshift/openshift-docs/pull/32472#discussion_r656556805

Reply:
Same comment as above.